ani-cli

```diff
@@ -150,54 +201,63 @@ get_video_quality() {
 	printf '%s' "$video_link"
 }
 
-dep_ch () {
-	for dep; do
-		if ! command -v "$dep" >/dev/null ; then
-			die "Program \"$dep\" not found. Please install it."
-		fi
-	done
+
+###############
+# Text output #
+###############
+
+err () {
+	# display an error message to stderr (in red)
+	printf "$c_red%s$c_reset\n" "$*" >&2
 }
 
-# get query
-get_search_query () {
-	if [ -z "$*" ]; then
-		printf "$c_cyan%s" "Search Anime: "
-		read -r query
-	else
-		query=$*
-	fi
+inf () {
+	# display an informational message (first argument in green, second in magenta)
+	printf "$c_green%s $c_magenta%s$c_reset\n" "$1" "$2"
 }
 
-# create history file
-[ -f "$logfile" ] || : > "$logfile"
+prompt () {
+        # prompts the user with message in $1-2 ($1 in blue, $2 in magenta) and saves the input to the variables in $REPLY and $REPLY2
+        printf "$c_blue%s$c_magenta%s$c_blue: $c_reset" "$1" "$2"
+        read -r REPLY REPLY2
+}
 
-#####################
-## Anime selection ##
-#####################
+menu_line_even () {
+	# displays an even (cyan) line of a menu line with $2 as an indicator in [] and $1 as the option
+	printf "$c_blue[$c_cyan%c$c_blue] $c_cyan%s$c_reset%s\n" "$2" "$1"
```

Comment:
What's the status on the shellcheck here?
Considering `printf` doesn't print a newline by default, couldn't you just make it two `printf`s to dodge the false positive?

Reply:
I see that it passed the shellcheck in its current form. Really similar code is in the master branch. Why would I change it?

Reply:
It didn't, we just ignore SC1087 and SC2059 currently

Reply:
Then I'll split it